api-tester/build.gradle

```diff
@@ -12,6 +12,7 @@ android {
             maxHeapSize = "1024m"
         }
     }
+    namespace 'com.revenuecat.api_tester_kotlin'
```

Comment:
Hmm it's weird this is needed now... Afaik, this should be the same as the package of the module by default... Was this due to the `R` file collision you mentioned?

Reply:
This was from an automatic migration to the latest AGP version. Maybe it's not mandatory?

Reply:
Oh interesting... Maybe it's needed if `applicationId` is not set? I thought it would automatically pick the package name and use that as the namespace but it's possible you need to set one or the other. I will take another look. In any case, I'm ok with this change!